src/main/java/mops/persistence/file/FileInfo.java

```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
     }
 
     /**
-     * Gives you FileInfoBuilder.
+     * Converts Byte to a String.
      *
-     * @return FileInfoBuilder
+     * @return Filesize with prefix.
      */
-    public static FileInfoBuilder builder() {
-        return new FileInfoBuilder();
+    public String getSizeString() {
+        final double kilobyte = 1024;
+        final double megabyte = kilobyte * 1024;
+        final double gigabyte = megabyte * 1024;
```

Comment:
```suggestion
        final double kilobyte = 1000;
        final double megabyte = kilobyte * 1000;
        final double gigabyte = megabyte * 1000;
```
1000 is used for Kilobyte, Megabyte and Gigabyte and is rounded.
1024 would be used for Kibibyte, Mebibyte and Gibibyte.
https://de.wikipedia.org/wiki/Byte#Vergleichstabelle

Reply:
???

Reply:
https://www.youtube.com/watch?v=TLgHOpYBnUM